### DISCUSSION TITLE
Locale in route url

### DISCUSSION THREAD

**Discussion by asnanmtakim** [codeigniter4/shield#726] [Q&A] [answered]
How add {locale} in route url? Like {locale}/login

**Comment by datamweb** [comment]
I had worked on this before, unfortunately I lost the branch. As far as I know, this possibility is not available by default in Shield.
To support this possibility, we have to make changes in the files.

https://forum.codeigniter.com/showthread.php?tid=86852&pid=407086#pid407086

**Comment by kenjis** [comment, marked as answer]
See https://codeigniter4.github.io/shield/customization/route_config/#use-locale-routes
